task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml

```diff
@@ -0,0 +1,303 @@
+          cpu: 250m
+          memory: 128Mi
+      securityContext:
+        capabilities:
+          add:
+            - SETFCAP
+    - name: sbom-syft-generate
+      image: registry.access.redhat.com/rh-syft-tech-preview/syft-rhel9:1.4.1@sha256:34d7065427085a31dc4949bd283c001b91794d427e1e4cdf1b21ea4faf9fee3f
+      workingDir: /var/workdir/source/source
+      volumeMounts:
+        - mountPath: /var/lib/containers
+          name: varlibcontainers
+      script: |
+        syft oci-dir:/var/lib/containers/rhtap-final-image --output cyclonedx-json=/var/workdir/source/sbom-cyclonedx.json
+      computeResources:
+        limits:
+          memory: 6Gi
+        requests:
+          memory: 6Gi
+    - name: merge-cachi2-sbom
+      image: quay.io/redhat-appstudio/sbom-utility-scripts-image@sha256:53a3041dff341b7fd1765b9cc2c324625d19e804b2eaff10a6e6d9dcdbde3a91
+      workingDir: /var/workdir/source
+      script: |
+        cachi2_sbom=./cachi2/output/bom.json
+        if [ -f "$cachi2_sbom" ]; then
+          echo "Merging contents of $cachi2_sbom into sbom-cyclonedx.json"
+          python3 /scripts/merge_cachi2_sboms.py "$cachi2_sbom" sbom-cyclonedx.json >sbom-temp.json
+          mv sbom-temp.json sbom-cyclonedx.json
+        else
+          echo "Skipping step since no Cachi2 SBOM was produced"
+        fi
+      securityContext:
+        runAsUser: 0
+    - name: inject-sbom-and-push
+      image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f
+      workingDir: /var/workdir/source
+      volumeMounts:
+        - mountPath: /var/lib/containers
+          name: varlibcontainers
+      script: |
+        #!/bin/bash
+        base_image_name=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.name"}}' "${IMAGE}" | cut -f1 -d'@')
+        base_image_digest=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.digest"}}' "${IMAGE}")
+        container=$(buildah from --pull-never "${IMAGE}")
+        buildah config -a org.opencontainers.image.base.name="${base_image_name}" -a org.opencontainers.image.base.digest="${base_image_digest}" "${container}"
+        buildah commit "${container}" "${IMAGE}"
+
+        status=-1
+        max_run=5
+        sleep_sec=10
+        for run in $(seq 1 $max_run); do
+          status=0
+          [ "$run" -gt 1 ] && sleep $sleep_sec
+          echo "Pushing sbom image to registry"
+          buildah push \
+            --tls-verify="${TLSVERIFY}" \
+            --digestfile "/var/workdir/source/image-digest" "${IMAGE}" \
+            "docker://${IMAGE}" && break || status=$?
+        done
+        if [ "$status" -ne 0 ]; then
+            echo "Failed to push sbom image to registry after ${max_run} tries"
+            exit 1
+        fi
+
+                                                                                 <"/var/workdir/source"/image-digest tee "$(results.IMAGE_DIGEST.path)"
```

Comment:
Something in the shell formatter does this, it's a generated file and this is mostly esthetics so I don't think it is worth troubleshooting, but yell at me if you think it matters...

Reply:
Is shfmt the tool used?

Reply:
We use https://github.com/mvdan/sh